package.json

```diff
@@ -0,0 +1,56 @@
+{
+	"name": "trader",
+	"version": "0.0.1",
+	"private": true,
```

Comment:
We might want to include 
```
{ "license": "UNLICENSED" }
```
cover our ass. I think changing that will be important in the future.